### PR TITLE
[Patch] Pool Mining Electronero via XMR-STAK as a config option

### PR DESCRIPTION
**Pull request by shopglobal**
Tested on Ubuntu 16.04 and it's performing nicely. 
This patch applies Electronero mining support in 3 commits. 
-Update to ReadMe.md (Add Electronero.org)
-Update to jconf.cpp (Add electronero, Cryptonight_v7 POW Algo and default pool)
-Update to pools.tpl (Add electronero to display options)

I mention some details regarding Electronero in #1599 

List of Electronero Mining Pools:
https://pool.electronero.org
https://uspool.electronero.org
https://afpool.electronero.org
https://etnxpool.org
https://etnxpool.com
http://etnx.thorshammer.cc
http://etnx.leafpool.com
http://etnx.magnificentpool.com
http://newpool.pw/electronero

### CODE REVIEW

**Comment by pwntechx** [comment]
i want  to use XMR-STAK with electronero as an option in the config

**Comment by psychocrypt** [comment]
Why do you temived so many other coints from the docs?
Currently everyone can miner electronero via the currency option
`cryptonight_v7`

**Comment by shopglobal** [comment]
Some kind of mistake I fixed it up in this PR

**Comment by shopglobal** [comment]
Please merge we have many support requests in our telegram https://t.me/electronero and https://t.me/etnxpool regarding this. We will also promote this on the main repo at launch and links back for links back guys let's support each other.

File: README.md [hunk 1]
```diff
@@ -45,13 +45,14 @@ Besides [Monero](https://getmonero.org), following coins can be mined using this
 - [Croat](https://croat.cat)
 - [Edollar](https://edollar.cash)
 - [Electroneum](https://electroneum.com)
+- [Electronero](https://electronero.org)
 - [Graft](https://www.graft.network)
 - [Haven](https://havenprotocol.com)
 - [Intense](https://intensecoin.com)
-- [IPBC](https://ipbc.io)
+- [IPBC](https://ipbc.io)
```
Comment (psychocrypt) [inline README.md]: please remove end of line white space chars

File: README.md [hunk 2]
```diff
@@ -45,13 +45,14 @@ Besides [Monero](https://getmonero.org), following coins can be mined using this
...
 - [Edollar](https://edollar.cash)
 - [Electroneum](https://electroneum.com)
+- [Electronero](https://electronero.org)
 - [Graft](https://www.graft.network)
 - [Haven](https://havenprotocol.com)
 - [Intense](https://intensecoin.com)
-- [IPBC](https://ipbc.io)
+- [IPBC](https://ipbc.io)	
 - [Karbo](https://karbo.io)
-- [Masari](https://getmasari.org)
```
Comment (psychocrypt) [inline README.md]: The order is alphabetic, please revert this change

**Comment by psychocrypt** [comment]
Please announce that the community must use `cryptonigt_v7`. We will  only add native coins if the pass 10 MH/s. the list will be to long if we add each little fork.

**Comment by shopglobal** [comment]
You're really making this difficult

**Comment by shopglobal** [comment]
I'll review the commits again in few hours.

**Comment by awakenvr** [comment]
ETNX as a default would help especially when setting up loads of rigs :)

**Comment by shopglobal** [comment]
Tony, I understand your concern about every new fork being added in general
as becoming something you may not prefer to oversee in general. However I
have received multiple complaints about the use of XMR-Stak returning "low
difficulty shares" (a common error produced by the nodejs pools indicative
of incorrect POW selection.) Adding Electronero to the config would solve
this issue. Making it easier for users to select. Also cryptonight_v7 only
applies to Electronero POW for some short period of time as the coin is
planned to upkeep with Monero developments.

Thanks for your consideration. If we must revisit upon main net launch then
we will have another go at this PR when the time is right.

**Comment by shopglobal** [comment]
This PR will be resubmitted after main net launch in 48 hours and I will take into account the requested changes before submitting. I have loved mining Monero with XMR-STAK for a long time.  We look forward to XMR-STAK being a preferred miner for the Electronero project.
